plain `cargo test` (without `--all-features`) errors

```
$ cargo test
   Compiling ...
error[E0432]: unresolved import `crate::measurements::make_laplace`
 --> src/accuracy/tail_bounds/test.rs:7:5
  |
7 |     measurements::make_laplace,
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ no `make_laplace` in `measurements`
```

Haven't looked into it, but my guess is that the code that defines `make_laplace` is guarded, but the test code does not have a corresponding guard. Might be considered a special case of
- #1343

--- a/.github/workflows/smoke-test.yml
+++ b/.github/workflows/smoke-test.yml
@@ -122,7 +122,10 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('rust/Cargo.toml') }}
           restore-keys: ${{ runner.os }}-cargo-
 
-      - name: Test
+      - name: Test (default)
+        run: cargo test --verbose
+
+      - name: Test (all-features)
         run: cargo test --verbose --all-features
 
       - name: Run docs example

--- a/rust/src/accuracy/tail_bounds/mod.rs
+++ b/rust/src/accuracy/tail_bounds/mod.rs
@@ -6,7 +6,7 @@ use crate::{
     traits::{InfAdd, InfCast, InfDiv, InfExp, NextFloat},
 };
 
-#[cfg(test)]
+#[cfg(all(test, feature = "contrib"))]
 mod test;
 
 /// Computes the probability of sampling a value greater than `t` from the discrete laplace distribution.

